### DISCUSSION TITLE
Kafka Snowflake - connector - parse the json/avro and store it in a table

### DISCUSSION THREAD

**Discussion by gvnsekhar** [community/community#56150] [Programming Help]
### Select Topic Area

Question

### Body

HI Team,
Currently we are able to store the data from the kafka topic, but we have a requirement to parse the message and store it in individual columns ( like relational database table) do you have any solution for this problem?

**Comment by tuves** [comment]
Thanks for posting in the GitHub Community, @gvnsekhar! 

We’ve moved your post to our new Programming Help 🧑‍💻 category, which is more appropriate for this type of discussion.

Please review our [guidelines](https://github.com/orgs/community/discussions/51384) about the Programming Help category for more information.
